testsuite/fuzzer/fuzz/fuzz_targets/move/aptosvm_publish_and_run.rs

```diff
@@ -205,21 +234,108 @@ fn filter_modules(input: &RunnableState) -> Result<(), Corpus> {
     Ok(())
 }
 
+pub fn code_publish_package_txn(
```

Comment:
We already have `aptos_cached_packages::aptos_stdlib::code_publish_package_txn` which does the same thing, can we reuse it and not introduce another function?

Reply:
I tried to avoid adding too much dependencies. but just checked and it's already in the dependency tree so I fixed it now